solutions/usaco-739.mdx

```diff
@@ -0,0 +1,142 @@
+---
+id: usaco-739
+source: USACO Silver 2017 US Open 
+title: Bovine Genomics
+author: Brad Ma
+---
+
+[Official Editorial (C++)](http://www.usaco.org/current/data/sol_cownomics_silver_open17.html)
+
+## Solution
+
+**Time Complexity** $\mathcal{O}(N^3)$
+
+<LanguageSection>
+  
+<JavaSection>
+  
+<Warning>
+This implementation uses sets, which has not been covered yet.
```

Comment:
```suggestion
This implementation uses sets, which have not been covered yet.
```

Reply:
Wait I dont even think this implementation uses sets